doctr/models/predictor/tensorflow.py

```diff
@@ -43,13 +42,22 @@ def __init__(
     def __call__(
         self,
         pages: List[Union[np.ndarray, tf.Tensor]],
+        rotate_document: bool = False,
```

Comment:
For consistency, I would suggest setting this as a class attribute in the constructor

As perhaps rename it to something rather descriptive like `straighten_pages`

Reply:
ok !